### PR TITLE
Update monero-core_nl.ts

### PR DESCRIPTION
**Pull request by Kortom-Tom**
i changed some translations and added some. mostly confirmed translations

### CODE REVIEW

**Comment by Kortom-Tom** [comment]
made some fixes to typos that killed the code.

**Comment by Kortom-Tom** [comment]
Hi i need help.
i don't see any code faults but its not passing the checks. I just translate. someone help.

**Comment by leonklingele** [comment]
Hi @Kortom-Tom, just take a look at the build log: https://build.getmonero.org/builders/monero-core-ubuntu-amd64/builds/1352/steps/compile/logs/stdio
Scroll down to the very bottom, it's complaining about

```
lrelease error: Parse error at ../translations/monero-core_nl.ts:2413:14: Opening and ending tag mismatch
```

Just remove the closing `message` tags (`</message>`) and it should build fine.

File: translations/monero-core_nl.ts [hunk 1]
```diff
@@ -2408,13 +2408,15 @@ Ringgrootte: </translation>
         <location filename="../main.qml" line="811"/>
         <location filename="../main.qml" line="831"/>
         <source>Payment proof check</source>
-        <translation type="unfinished"></translation>
+        <translation>Betaalbewijs gecheckt</translation>
+    </message>
```
Comment (leonklingele) [inline translations/monero-core_nl.ts]: Remove this

File: translations/monero-core_nl.ts [hunk 2]
```diff
@@ -2408,13 +2408,15 @@ Ringgrootte: </translation>
...
+    </message>
```
Comment (leonklingele) [inline translations/monero-core_nl.ts]: Remove this line

File: translations/monero-core_nl.ts [hunk 3]
```diff
@@ -2465,22 +2467,23 @@ Ringgrootte: </translation>
...
         <source>Cancel</source>
-        <translation type="unfinished">Annuleren</translation>
+        <translation>Annuleren</translation>
     </message>
     <message>
         <location filename="../main.qml" line="1174"/>
         <source>Password changed successfully</source>
-        <translation type="unfinished"></translation>
+        <translation>Wachtwoord succesvol veranderd</translation>
+    </message>
```
Comment (leonklingele) [inline translations/monero-core_nl.ts]: And this

File: translations/monero-core_nl.ts [hunk 4]
```diff
@@ -2505,7 +2508,8 @@ Ringgrootte: </translation>
     <message>
         <location filename="../main.qml" line="1599"/>
         <source>Tap again to close...</source>
-        <translation type="unfinished"></translation>
+        <translation>Opnieuw klikken om te sluiten.</translation>
+    </message>
```
Comment (leonklingele) [inline translations/monero-core_nl.ts]: And that one too

File: translations/monero-core_nl.ts [hunk 5]
```diff
@@ -2520,7 +2524,8 @@ Ringgrootte: </translation>
     <message>
         <location filename="../main.qml" line="1615"/>
         <source>Stop daemon</source>
-        <translation>Stop node</translation>
+        <translation>Stop daemon</translation>
+    </message>
```
Comment (leonklingele) [inline translations/monero-core_nl.ts]: Don't forget about this little fella

Comment (Kortom-Tom) [reply translations/monero-core_nl.ts]: thanks!!

**Pull request closed without merging**